### PR TITLE
PFM-ISSUE-7768 - [bug] grab jfrog creds from env for circle CI

### PR DESCRIPTION
**Pull request by smasala**

### CODE REVIEW

File: src/model/NPMResolver.ts [hunk 1]
```diff
@@ -329,22 +329,30 @@ export class NPMResolver {
...
 
-        if (!gradleProps) {
-            console.error(cred`✗`, 'Gradle Properties not found');
+        if (!cleanNpmrcPath) {
+            console.error(cred`✗`, 'No user npmrc found');
+            return;
+        }
+
+        if(process.env.ENV_CPLACE_ARTIFACTORY_ACTOR && process.env.ENV_CPLACE_ARTIFACTORY_TOKEN) {
+            this.writeNPMRC(registry, cleanNpmrcPath, Buffer.from(`${process.env.ENV_CPLACE_ARTIFACTORY_ACTOR}:${process.env.ENV_CPLACE_ARTIFACTORY_TOKEN}`).toString('base64'), process.env.ENV_CPLACE_ARTIFACTORY_ACTOR);
```
Comment (Kaltenbach) [inline src/model/NPMResolver.ts:343]: For non docker ci-s (like a Jenkins could be), the `const isConfigured: boolean = currentNpmrcConfig.includes(registry);` check could also make sense here.

**Review by Kaltenbach — CHANGES_REQUESTED** [review]
I don't know much about node js but looks like it does what it should 😄

**Review by Kaltenbach — APPROVED** [review]
:+1:

**Comment by rishoocplace** [comment]
Tested on Windows, seems to work good 👍

**Review by slaven3kopic — APPROVED** [review]

**Pull request merged**